packages/components/src/select-control/index.js

```diff
@@ -25,15 +25,10 @@ const initialState = { isExpanded: false, isFocused: false, query: '' };
 export class SelectControl extends Component {
 	constructor( props ) {
 		super( props );
-
-		const { selected, options } = props;
 		this.state = {
 			...initialState,
 			searchOptions: [],
-			selectedIndex:
-				selected && options?.length
-					? options.findIndex( ( option ) => option.key === selected )
-					: 0,
+			selectedIndex: -1,
```

Comment:
This seems to break `SelectControl`s that are supplied a `selected` prop. I think all we need to change from the original code is the `selectedIndex` from `0` to `-1`.

Reply:
Sorry Jeff I don't get this one, this change _is from `0` to `-1`_ from the original code. I've also tested a `SelectControl` with `selected` prop given, and seems working fine with mine. If it's possible, maybe you can help to provide an example where it breaks? :pray:

I'll make an update to the storybook to include an example with `selected` as well.

Reply:
Ah, I misunderstood your original suggestion 🤦 . Changes made in 0890a7b